test_googleMapsApiModule.py

```diff
@@ -0,0 +1,36 @@
+import googleMapsApiModule
+import config
+import pytest
+from googlemaps.exceptions import ApiError
+ 
+class TestClass:
+    def test_direction(self):
+        result = googleMapsApiModule.direction('paris','brussels')
+        assert result == "https://www.google.com/maps/dir/?api=1&origin=paris&destination=brussels"
+
+    def test_invalid_direction(self):
+        with pytest.raises(ApiError):
+            result = googleMapsApiModule.direction('kjajw','qwiuq')
+            assert result == "https://www.google.com/maps/dir/?api=1&origin=kjajw&destination=qwiuq"
+
+    def test_geocoding(self):
+        result = googleMapsApiModule.geocoding('denver')
+        assert result == "https://www.google.com/maps/search/?api=1&query=denver"
+
+    def test_Invalid_geocoding(self):
```

Comment:
Let's have a bit more standardized naming no?

```python3
def test_{function_name}_with_{valid|invalid}_input():
```

Reply:
I have changed the names.